package.json

```diff
@@ -55,9 +55,6 @@
     "readable-stream": "^3.6.0",
     "uuid": "^8.3.2"
   },
-  "peerDependencies": {
-    "better-sqlite3": ">=8.7.0"
-  },
```

Comment:
this should be moved to `dependencies` right?

Reply:
I thought really hard about doing this and I guess I didn't! Done in a180c1319f154afda4eb1c976d7c6dc8a3f4a8ce.